addons/input-overlay/input-overlay.gd

```diff
@@ -16,15 +16,18 @@ func _enter_tree():
 	# layout
 	main_screen.resized.connect(position_widget)
 	position_widget()
-
+	scale_widget()
 
 func position_widget():
 	var rect: Rect2 = main_screen.get_global_rect()
 	panel.position.x = rect.position.x
-	panel.position.y = rect.position.y + rect.size.y - panel.get_rect().size.y
+	panel.position.y = rect.position.y + rect.size.y - (panel.get_rect().size.y * conf.scale)
```

Comment:
Maybe we cancall `scale_widget` as a first instruction in `position_widget` and avoid multiplying by conf.scale? 
In this way the widget can be scaled even without rebooting Godot (or toggling the addon).

During development I find useful to tweak values in the conf, then resizing the main screen to see changes. 
In the future it would be nice also to detect resource prop changed events and automatically trigger `position_widget` or related functions so that we can see updates in real time.